examples/apeworx_everlasting_bots.py

```diff
@@ -262,91 +267,92 @@ def get_argparser() -> argparse.ArgumentParser:
 def get_config() -> simulators.Config:
     """Set config values for the experiment"""
     args = get_argparser().parse_args()
-    config = simulators.Config()
-    config.log_level = output_utils.text_to_log_level(args.log_level)
-    config.log_filename = "everlasting_bots"
-    config.title = "everlasting bot demo"
+    _config = simulators.Config()
```

Comment:
why underscore?

Reply:
lots of variable renamed to _variable to avoid, for example: `Redefining name 'position_duration_seconds' from outer scope (line 517)`

Reply:
so jsut a pylint thing, which would be avoided if we excluding it from linting... is that consistent with our other examples? maybe we should just exclude the whole `examples` folder

Reply:
Ahh ok, right. This script defines some variables in the top-most scope (`__name__ == "__main__"`). Any variable defined there should be unique from variables that exist inside of functions. This is bc any variable defined in the global scope should still technically be accessible in the function. i.e. if we didn't overwrite `config` in get_agents, then we would be able to query it. Adding an underscore is typically used for "private" variables/functions, so I see where you're going with that now. However, I think it is more clear to rename the variable in the global context to be more specific to an instantiated config. e.g. `sim_config`, meaning "the config for this particular sim" and leaving the variable name as `config` throughout the rest of the functions. I think this makes the distinction between the config used in execution vs an internal variable more clear.

Reply:
After writing that I see you also ran into the problem with `sim_agents` and `sol_agents`. With that in mind, and also keeping in mind that this script itself is probably not going to survive long in the repo (it's janky, will be integrated soon, was never intended to have staying power, etc), I'll revise the recommendation to just leave it alone. `_` is fine for now.